libzhlgen/test/functions/ELF_amd64/1.6.13/HackingSystem.zhl

```diff
@@ -2,3 +2,5 @@
 cleanup __amd64 void HackingSystem::BlowHackingDrone(HackingSystem *this);
 "55534889fb4883ec38e8????????80bb7002000000":
 cleanup __amd64 void HackingSystem::OnLoop(HackingSystem *this);
+"5380bf44020000????89fb75??31c05bc30f1f8000000000e8????????83f8??75??0fb683b0050000":
```

Comment:
Wildcard's more than needed, this works but so does:
`"5380bf44020000004889fb750b31c05bc30f1f8000000000e8????????83f8ff"`

Although it is cheating a little because I know from comparing both that I can skip the wildcarding of the `JNZ` so:
`"5380bf44020000004889fb75??31c05bc30f1f8000000000e8????????83f8ff"`
Is technically the most correct of just skipping the addresses with the `A` button in Ghidra's instruction pattern search

This one doesn't have to change, this is just informational for you.

Reply:
Fixed